.gitignore

```diff
@@ -5,4 +5,6 @@ package-lock.json
 .next
 .env.local
 yarn-error.log
-coverage
\ No newline at end of file
+coverage
+.yarn
+.yarnrc.yml
```

Comment:
why is this changed?

Reply:
On `yarn install` or `yarn` it created **.yarn** directory with few script files 
and **.yarnrc.yml** which is not required. 
So, I added to gitignore.

Reply:
@RitikJaiswal75 also faced the same issue. He can help you out with fixing it.

Reply:
Reverted.